schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals(
+    id INT,
```

Comment:
- [ ] [OPTIONAL] You can use the following code to set the `id` column as a primary key:
```SQL
id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY
```